source/app/blueprints/rest/manage/manage_cases_routes.py

```
@@ -31,14 +38,23 @@
 from app.blueprints.rest.endpoints import endpoint_deprecated
 from app.datamgmt.alerts.alerts_db import get_alert_status_by_name
 from app.datamgmt.case.case_db import get_case
+from app.datamgmt.client.client_db import get_client_list
```

Ruff (0.8.2)

`app.datamgmt.client.client_db.get_client_list` imported but unused

Remove unused import: `app.datamgmt.client.client_db.get_client_list`

(F401)